src/js/lib/youtube.js

```diff
@@ -21,6 +21,15 @@ export function pimpYouTubePlayer(videoId, node, height, width) {
             function playerReady() {
                 youTubePlayer.playVideo();
             }
+
+            var chapterBtns = [].slice.call(document.querySelectorAll('.docs--chapters li'));
```

Comment:
We're going to have [two versions of the same thing soon](https://github.com/guardian/docs-interactive-template/pull/21/files#diff-691ff2b74e21c3011d7cec25c48f6146R48). We should abstract it out into a module that's shared to keep things DRY.